src/lib/api.ts

```diff
@@ -71,7 +86,7 @@ export const loadGroup: Load = async (args) => {
 };
 
 export const loadGroups: Load = async ({ fetch, params, url: urlParam }) => {
-	const res = await fetch(url(`group_list?all_fields=true&${urlParam.searchParams}`));
+	const res = await fetch(apiUrl(`group_list?all_fields=true&${urlParam.searchParams}`));
```

Comment:
Thanks for your care with this. I did not make the name change myself, but let the Visual Studio Code deal with it checking for all references. I was afraid of this name change myself since 'url' is a quite general name. I double checked and tested, so it is all fine as far as I can tell.